runelite-client/src/test/java/net/runelite/client/plugins/cannon/CannonPluginTest.java

```diff
@@ -204,6 +204,7 @@ public void cannonDestroyedSetupNormalCannonballs()
 		assertEquals(30, plugin.getCballsLeft());
```

Comment:
after this commit these arnt really testing the inventory diff system since the message will add 5 and 25 to equal 30 regardless of what the inventory diff decides the contents are